cmd/dashboard/main.go

```diff
@@ -52,39 +70,45 @@ func main() {
 
 	pipelineClient, err := clientset.NewForConfig(cfg)
 	if err != nil {
-		logging.Log.Errorf("error building pipeline clientset: %s", err.Error())
-	} else {
-		logging.Log.Info("Got a pipeline client")
+		logging.Log.Errorf("Error building pipeline clientset: %s", err.Error())
 	}
 
 	k8sClient, err := k8sclientset.NewForConfig(cfg)
 	if err != nil {
-		logging.Log.Errorf("error building k8s clientset: %s", err.Error())
-	} else {
-		logging.Log.Info("Got a k8s client")
+		logging.Log.Errorf("Error building k8s clientset: %s", err.Error())
 	}
 
 	resource := endpoints.Resource{
 		PipelineClient: pipelineClient,
 		K8sClient:      k8sClient,
 	}
 
+	// Any functional changes to the the router mux should be reflected in endpoints/test-utils.test_go/func dummyServer()
```

Comment:
As I understand the [docs](https://golang.org/pkg/cmd/go/internal/test/#pkg-variables) for this, anything with the `_test` package/file suffix is not subject to the dependency relations of the actual directory/package. This way, you are able to prevent certain cyclical dependencies without needing to make an entirely new dir/package just to test.

Reply:
I moved this to it's own package since the new router package has tests that also uses it.